src/main/java/com/github/instagram4j/instagram4j/requests/users/UsersInfoRequest.java

```diff
@@ -2,21 +2,20 @@
 
 import com.github.instagram4j.instagram4j.requests.IGGetRequest;
 import com.github.instagram4j.instagram4j.responses.users.UserResponse;
-
 import lombok.AllArgsConstructor;
 
 @AllArgsConstructor
 public class UsersInfoRequest extends IGGetRequest<UserResponse> {
```

Comment:
A note here, the library mostly follows the Google style Java coding conventions and uses 4 spaces in place of tabs. 
I personally do not have an issue between the two, but for consistency, please reformat code to use 4 spaces instead of tabs.